lib/trento_web/controllers/session_controller.ex

```diff
@@ -90,41 +89,9 @@ defmodule TrentoWeb.SessionController do
     end
   end
 
-  operation :show,
```

Comment:
question:
this was superseded by the `/api/profile` call, right?

Reply:
yep, indeed